### ISSUE TITLE
Node Color

### ISSUE THREAD

**Issue by heboet** [israelpohl/vizlaw_frontend#5]
Optimizing the node color (e.g. special color for most recent decision). Other color features?

**Comment by heboet** [comment]
The younger the darker...

```
const darken = (n) => `rgba(208, 65, 71, ${n})`

var timefromnow = Math.abs(moment(node.date).diff(moment(), "years"));
node.color = darken(1/(timefromnow*2))
```

**Comment by HappyFi** [comment]
Color depends on court level. We should add blue for EuGH decisions.

**Comment by heboet** [comment]
Of course, this would increase distinguishability, but it might interfere with a harmonious design and appearance.

**Comment by HappyFi** [comment]
I don't think that only red colors will help to improve the visibility. We should use (try our) more colors. If you want to work fast with this tool you can't differ between light light light red (AG), light light red (LG), light red (OG) , red (BG), darker red (EG).

**Comment by heboet** [comment]
True. We might even use different shapes for different courts. But too much of different graphical information might cause confusion instead of visual clarity.
